client/__tests__/e2e/puppeteerUtils.js

```diff
@@ -16,7 +16,7 @@ const puppeteerUtils = (page) => ({
   },
 
   async getAllByClass(testClass) {
-    return page.$$eval(`[data-testclass=${testClass}]`, (eles) => eles.slice());
+    return page.$$(`[data-testclass=${testClass}]`);
```

Comment:
Ah, if I'm trying to just return the elements, there's no need for the `eval` here. Thanks!